docs/setting-up/client/postgresql.md

```diff
@@ -309,6 +321,24 @@ where:
 - `USER`: Database user allowed to connect via TLS. Should match the common name (CN) used in the client certificate.
 - `SERVICE`: Name to give to the service within PMM.
 
+#### Automatic discovery via CLI
+
+Starting with PMM 2.41.0, there is a new flag in `pmm-admin` to limit Auto-discovery:
+
+`--auto-discovery-limit=XXX`
+
+If the limit is equal to or greater than the number of databases, Automatic-discovery is enabled. If the limit is exceeded, then Automatic-discovery is disabled.
+
+In case of a default value of zero, `Automatic- discovery` is enabled with no limit.
+
+
+Example:
+
+If you set the limit to 10 and your PostgreSQL instance has 11 databases, Automatic-discovery will be disabled.
+
+`pmm-admin add postgresql --username="pmm-agent" --password="pmm-agent-password" --auto-discovery-limit=18`
```

Comment:
With example above there should be --auto-discovery-limit=10

Reply:
ok